### PR TITLE
Add PushStarter template

### PR DESCRIPTION
**Pull request by corinnekrych**
Not to merge.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -66,6 +66,30 @@
...
+        "type": "default",
+        "icon": "icon-plane",
+        "category": "Samples",
+        "screenshots": [],
+        "appTemplates": [
+          {
+            "id": "pushstarter_ios_app",
+            "name": "Simple Push iOS App",
+            "type": "client_native_ios",
+            "repoUrl": "git://github.com/corinnekrych/pushstarter-ios-app.git",
```
Comment (edewit) [inline global.json]: don't you want this to point to your local fork, but put it into https://github.com/feedhenry-templates

**Comment by corinnekrych** [comment]
@edewit Done, I'm now pointing to feddhenry-template organization. 
@danielpassos @edewit add your windows/android templates there.

File: global.json [hunk 2]
```diff
@@ -66,6 +66,52 @@
...
+            "name": "Simple Push Android App",
+            "type": "client_native_android",
+            "repoUrl": "git@github.com:danielpassos/pushstarter-android-app.git",
+            "githubUrl": "git@github.com:danielpassos/pushstarter-android-app.git",
+            "repoBranch": "refs/heads/master",
+            "icon": "icon-android",
+            "category": "Sample Apps",
+            "screenshots": []
+          },
+          "hello_world_mbaas_instance"
```
Comment (wei-lee) [inline global.json]: Try replace this line with

```json
{
  "id": "hello_world_mbaas_instance",
  "type": "cloud_nodejs"
}
```

**Comment by matzew** [comment]
@corinnekrych @edewit what's the status here ?

**Comment by matzew** [comment]
JIRA for this PR =-> https://issues.jboss.org/browse/FH-277

**Comment by corinnekrych** [comment]
@matzew @danielpassos  checking the config file we're missing final repo URL for Android:
https://github.com/corinnekrych/template-config/blob/ups.template.app/global.json#L105
Apart from that i think we can merge it.

**Comment by danielpassos** [comment]
@corinnekrych Just sent a [PR](https://github.com/corinnekrych/template-config/pull/4) to your repo

**Comment by matzew** [comment]
is this ready to merge ?

**Comment by matzew** [comment]
@edewit @corinnekrych @jasonmadigan is this good to merge?

**Comment by edewit** [comment]
@matzew :+1:

**Comment by jasonmadigan** [comment]
:-1: build failing currently:

```
Running "gitclone:pushstarter_ios_app" (gitclone) task
Warning: Cloning into 'tmp/pushstarter_ios_app'...
fatal: unable to connect to github.com:
github.com: Servname not supported for ai_socktype Use --force to continue.
```

**Comment by edewit** [comment]
The error seems to be the build not the PR

**Comment by jasonmadigan** [comment]
@edewit yeah, still needs to be fixed though (not 100% sure what's wrong without digging into it). Without a working PR build, you can't update fhcap with an updated `fh-template-apps` component, so this would never actually make it into production

**Comment by jasonmadigan** [comment]
built this locally, same error - there's some small issues with repoUrl URIs. patch to fix:

```
diff --git a/global.json b/global.json
index d3f3327..3246fae 100644
--- a/global.json
+++ b/global.json
@@ -79,7 +79,7 @@
             "id": "pushstarter_ios_app",
             "name": "Simple Push iOS App",
             "type": "client_native_ios",
-            "repoUrl": "git://github.com:feedhenry-templates/pushstarter-ios-app.git",
+            "repoUrl": "git://github.com/feedhenry-templates/pushstarter-ios-app.git",
             "githubUrl": "https://github.com/feedhenry-templates/pushstarter-ios-app.git",
             "repoBranch": "refs/heads/master",
             "icon": "icon-apple",
@@ -90,7 +90,7 @@
             "id": "pushstarter_windows_app",
             "name": "Simple Push Windows App",
             "type": "client_native_windowsuniversal",
-            "repoUrl": "git://github.com:feedhenry-templates/pushstarter-windows-app.git",
+            "repoUrl": "git://github.com/feedhenry-templates/pushstarter-windows-app.git",
             "githubUrl": "https://github.com/feedhenry-templates/pushstarter-windows-app.git",
             "repoBranch": "refs/heads/master",
             "icon": "icon-windows8",
@@ -101,7 +101,7 @@
             "id": "pushstarter_android_app",
             "name": "Simple Push Android App",
             "type": "client_native_android",
-            "repoUrl": "git@github.com:feedhenry-templates/pushstarter-android-app.git",
+            "repoUrl": "git://github.com/feedhenry-templates/pushstarter-android-app.git",
             "githubUrl": "https://github.com/feedhenry-templates/pushstarter-android-app.git",
             "repoBranch": "refs/heads/master",
             "icon": "icon-android",
```

**Comment by jasonmadigan** [comment]
I would suggest squashing these into one commit - there's some merge commits in the middle of the tree though, so might make it a little harder - may want to just create a new PR

**Comment by corinnekrych** [comment]
closing it in favor of https://github.com/feedhenry/template-config/pull/62

**Pull request closed without merging**